modules/graphstudio/pages/map-data-to-graph.adoc

```diff
@@ -19,34 +19,34 @@ The main steps are
 . Map data file columns to vertex/edge fields
 . Publish data mapping
 
-== Select a data source
+== Select a data source type
 
-GraphStudio supports loading data from either local files on the TigerGraph server or files stored on Amazon S3.
+GraphStudio supports loading data from local files on the TigerGraph server, from files stored on Amazon S3, or from files stored on Google Cloud Storage (GCS).
 
-Click the data file type selector button on the banner of the Add Data File window, and choose either File or S3 from the list:
+Click the Add Data File button image:add_data_source_btn.png[] in the toolbar to add data files.
 
-image::Screen Shot 2020-01-29 at 2.15.45 PM.png[]
+image::choose-a-data-source-type.png[]
 
-* If you select *File*, no more configuration is needed. Skip the sections for external sources and go to  btn:[Map Data To Graph].
-* If you select *S3*, then read the section <<_create_s3_data_source>>.
+* If you select *Local File*, no more configuration is needed. Skip the sections for external sources and go to btn:[Map Data To Graph].
+* For *AWS S3*, skip to the section <<_create_s3_data_source>>.
+* For *Google Cloud Storage*, skip to the section <<_create_gcs_data_source>>.
 
 == Add data files
 
+A data file is a file containing structured data to be loaded into the graph, creating vertex and/or edge instances.
+After specifying your data source type, the next step is to specify your data files.
+
 This section contains subsections for each data source.
 
 * Local File System - <<_add_local_data_file>>
 * AWS S3 - <<_create_s3_data_source>>
+* GCS - <<_create_gcs_data_source>>
```

Comment:
It looks like the link here is still not working yet. It also looks like this:
<img width="309" alt="Screen Shot 2022-05-12 at 1 41 13 PM" src="https://user-images.githubusercontent.com/81188193/168164261-5e4c397c-c97c-4ecc-9fce-b1fd8e638c57.png">